tests/unit/test-extension-runners/test.firefox-desktop.js

```diff
@@ -285,6 +285,7 @@ describe('util/extension-runners/firefox-desktop', () => {
     assert.equal(install.asProxy, true);
     assert.equal(install.manifestData.applications.gecko.id,
                  manifestData.applications &&
+                 manifestData.applications.gecko &&
```

Comment:
This seems to be the pattern to silence flow here, but it feels wrong. Should we maybe just test for the exact string here rather than optional chaining these properties? (same for the test.sign.js version)